aws/resource_aws_servicecatalog_portfolio.go

```diff
@@ -41,9 +41,8 @@ func resourceAwsServiceCatalogPortfolio() *schema.Resource {
 				Computed: true,
 			},
 			"name": {
-				Type:         schema.TypeString,
-				Required:     true,
-				ValidateFunc: validation.StringLenBetween(1, 20),
+				Type:     schema.TypeString,
+				Required: true,
```

Comment:
Do we want to enforce [documented](https://docs.aws.amazon.com/servicecatalog/latest/dg/API_CreatePortfolio.html#servicecatalog-CreatePortfolio-request-DisplayName) max length of `100`?

Reply:
I have mixed feelings about enforcing these - mostly against. AWS changes them (across the API) regularly and so we're enforcing the wrong limits much of the time. But, it may save a long wait in certain situations. Most of the time though, AWS responds quickly if validation is failing.